skills/unraid/references/endpoints.md

```
@@ -0,0 +1,49 @@
+
+## Monitoring (2)
+15. **`notifications`** - System alerts
+16. **`upsDevices`** - UPS battery status
+
+## User & Auth (4)
+17. **`me`** - Current user info
+18. **`owner`** - Server owner
+19. **`isSSOEnabled`** - SSO status
+20. **`oidcProviders`** - OIDC providers
+
+## API Management (2)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)